### PR TITLE
HW

### PR DESCRIPTION
**Pull request by mcc33**
HW # 6 :)

### CODE REVIEW

**Comment by mcc33** [comment]
Added the Thermostat HW

File: MikeCassaro-mcc33/hw_6/rise_of_robots.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    else
+      "I have no empathy, I am a robot."
+    end
+  end
+
+end
+
+#nana = Robot.new("Nana", "Super-Android", "New Orleans")
+#puts nana.flying_skills
+Robot.make_robots(10)
```
Comment (chylaudes) [inline MikeCassaro-mcc33/hw_6/rise_of_robots.rb]: Sweet! :+1: Great use of conditionals.

File: MikeCassaro-mcc33/hw_7/thermostat.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+attr_accessor :degrees
+
+  def initialize(temp, target)
+    @degrees = temp #degree instanse variable set equal to the passed temp argument
+    @target = target #target instance varaible is set equal to the passed temp argument.
+  end
+
+#self.target, @target and target are all the same inside of this instance method, beause @instance_variables live within the class.
+  def turn_off_turn_on
+      if degrees < @target
```
Comment (chylaudes) [inline MikeCassaro-mcc33/hw_7/thermostat.rb]: 'degrees' also works, but recognizing as an instance variable just to make it consistent is also best practice
```
if @degrees < @target
```

File: MikeCassaro-mcc33/hw_7/thermostat.rb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+#If the temp is above the target temp, create a method that gets desired temp
+
+class Thermostat
+
+attr_accessor :degrees
+
+  def initialize(temp, target)
+    @degrees = temp #degree instanse variable set equal to the passed temp argument
+    @target = target #target instance varaible is set equal to the passed temp argument.
+  end
```
Comment (chylaudes) [inline MikeCassaro-mcc33/hw_7/thermostat.rb]: I really like your comment here!  Recognizing that your argument sets the value of each of your instance variables.  :+1:  :cow:

**Comment by chylaudes** [comment]
Awesome!  Great use of both class and instance methods. Just one naming convention error, but I am just being picky.  Also your comments are great, especially when you're trying to put into your own words complex ruby concepts.

**Pull request merged**